docs/index.rst

```diff
@@ -48,18 +48,18 @@ The purpose of this documentation is to list and protocol all the information ab
    components
 
 
-If any of these documents helped you in some way or another for one of your projects then please credit us and/or include a direct link to this document.
+If any of these documents helped you in some way or another for one of your projects, please credit us and/or include a direct link to this document!
 
 .. warning ::
 
-	LEGO is in no way affiliated with the content of this and the aforementioned documents.
-	Furthermore the creators of the just mentioned documents are not associated or involved with LEGO.
+	Neither the original documentation nor this read-the-docs port is associated or involved with The Lego Group.
```

Comment:
```suggestion
	Neither the original documentation nor this read-the-docs port is associated or involved with The LEGO Group.
```

I think we want to go along with their requests to capitalize it 🙈😅

Reply:
Unintentional -- Wikipedia said it wasn't capitalized but it looks like their brand guidelines page says it is LOL